jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/bigquery/BigQuerySqlGenerationVisitor.java

```diff
@@ -0,0 +1,27 @@
+package com.exasol.adapter.dialects.bigquery;
+
+import com.exasol.adapter.*;
+import com.exasol.adapter.dialects.*;
+import com.exasol.adapter.sql.*;
+
+import java.util.*;
+
+public class BigQuerySqlGenerationVisitor extends SqlGenerationVisitor {
+    public BigQuerySqlGenerationVisitor(final SqlDialect dialect, final SqlGenerationContext context) {
+        super(dialect, context);
+    }
+
+    @Override
+    public String visit(final SqlOrderBy orderBy) throws AdapterException {
+        final List<String> sqlOrderElement = new ArrayList<>();
+        for (int i = 0; i < orderBy.getExpressions().size(); ++i) {
+            String elementSql = orderBy.getExpressions().get(i).accept(this);
```

Comment:
1. Move down one line.
1. Make it final.
1. Replace the if with the `? ... : ...` notation.

Reply:
Can't fe final. because we change it later

Reply:
fixed